TACC/Core-Portal
# task/FP-895: Re-style site search file listing as a table

## Overview: ##

## Related Jira tickets: ##

* [FP-895](https://jira.tacc.utexas.edu/browse/FP-895)

## Summary of Changes: ##

## Testing Steps: ##
1. Perform a search in Site Search 
2. Click on a file result and verify that it opens a preview (for a file) or navigates (for a folder).

## UI Photos:

## Notes: ##
